themes/openy_themes/openy_rose/templates/node/node--branch--teaser.html.twig

```diff
@@ -99,18 +102,14 @@
   {{ content.field_location_state }}
   {{ content.field_location_area }}
 
-  {% if display_submitted %}
-    <footer class="node__meta">
-      {{ author_picture }}
-      <div{{ author_attributes.addClass('node__submitted') }}>
-        {% trans %}Submitted by {{ author_name }} on {{ date }}{% endtrans %}
-        {{ metadata }}
-      </div>
-    </footer>
-  {% endif %}
-
   <div{{ content_attributes.addClass('node__content') }}>
-    {{ content|without('field_location_area', 'field_location_state') }}
+    {{ content.field_location_address }}
+    {{ content.field_location_phone }}
+
+    {% if directions_url is not empty %}
+      <a href="{{ directions_url }}" title="{{ directions_title }}">{{ directions_title }}</a>
```

Comment:
The <a> tag doesn't have to be created manually. There is a link() function to do it. See https://www.drupal.org/docs/8/theming/twig/functions-in-twig-templates